apps/chat/src/components/Common/ConfirmDialog.tsx

```diff
@@ -10,6 +12,9 @@ interface Props {
   description?: string;
   confirmLabel: string;
   cancelLabel?: string | null;
+  textClasses?: string;
+  buttonsClasses?: string;
+  descriptionClasses?: string;
```

Comment:
You don't need this - just change confirmation modal and it will be applied to all such confirmation modals